ledger/participant-integration-api/src/main/scala/platform/store/backend/common/ContractStorageBackendTemplate.scala

```diff
@@ -100,10 +64,45 @@ trait ContractStorageBackendTemplate extends ContractStorageBackend {
     if (ids.isEmpty) {
       Failure(emptyContractIds)
     } else {
-      def lookup(id: ContractId): Option[Option[Instant]] =
-        maximumLedgerTimeSqlLiteral(id).as(instantFromMicros("ledger_effective_time").?.singleOpt)(
+      def lookup(id: ContractId): Option[Option[Instant]] = {
```

Comment:
Why did you inline `maximumLedgerTimeSqlLiteral` here? Could you have just modified `maximumLedgerTimeSqlLiteral` instead?

Reply:
The `maximumLedgerTimeSqlLiteral` was introduced in my previous PR #10888 in order to create a different implementation for H2. Currently there's a single implementation, so I reverted the `lookup()` method to have an inlined query.